internal/cli/apps.go

```diff
@@ -173,6 +174,34 @@ var CmdAppLogs = &cobra.Command{
 	},
 }
 
+// CmdAppExec implements the command: epinio apps exec
+var CmdAppExec = &cobra.Command{
+	Use:   "exec NAME",
+	Short: "creates a shell to the application",
+	Args:  cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cmd.SilenceUsage = true
+
+		client, err := usercmd.New()
+		if err != nil {
+			return errors.Wrap(err, "error initializing cli")
+		}
+
+		// TODO: "--instance" ? to define the app instance (the pod)
+		// follow, err := cmd.Flags().GetBool("follow")
+		// if err != nil {
+		// 	return errors.Wrap(err, "error reading option --follow")
+		// }
```

Comment:
Can we remove this comment?

Reply:
Extracted to a new issue: https://github.com/epinio/epinio/issues/1151

removing TODO